src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -4,7 +4,7 @@ Dashboard Developer Documentation
 Frontend Development
 --------------------
 
-Before you can start the dashboard from within a development environment,  you
```

Comment:
Is this related to this PR?

Reply:
I only removed the unnecessary blank. Should I open a separate pull request for that?

Reply:
I only removed the unnecessary blank. Should I open a separate pull request for that?

Reply:
I think it's fine to leave it in here.